ui/hooks/gasFeeInput/useTransactionFunctions.js

```diff
@@ -11,13 +11,18 @@ import { updateTransaction as updateTransactionFn } from '../../store/actions';
 export const useTransactionFunctions = ({
   defaultEstimateToUse,
   gasFeeEstimates,
-  gasLimit,
+  gasLimit: gasLimitInTransaction,
   transaction,
 }) => {
   const dispatch = useDispatch();
 
   const updateTransaction = useCallback(
-    (estimateUsed, maxFeePerGas, maxPriorityFeePerGas) => {
+    (
+      estimateUsed,
+      maxFeePerGas,
+      maxPriorityFeePerGas,
+      gasLimit = gasLimitInTransaction,
```

Comment:
[nit] What are your thoughts on using an object to represent these arguments? It might be easier to work with since now you have four arguments and one of them is optional.

Reply:
Good point, I made the change